DEPS

```diff
@@ -40,6 +40,7 @@ deps = {
     "url": "https://github.com/ronaldoussoren/macholib.git@36a6777ccd0891c5d1b44ba885573d7c90740015",
     "condition": "checkout_mac",
   },
+  "third_party/adblock-resources/src": "https://github.com/brave/adblock-resources@master"
```

Comment:
@atuchin-m actually that would defeat the purpose here. We always want the most recent one at sync time

Reply:
@bridiver using master here makes the builds non-reproducable.

When I build some old revision I would like to have the matching old list, not the recent one.

Reply:
we definitely need reproducible builds. We can't for example respin a build that QA already tested for a minor fix and then have an unexpected breakage.  We would just need some kind of automation that creates a pull request periodically that updates to the latest hash. And we can merge it.

Reply:
is this really that different than using components in general? The exact same version of Brave can behave differently depending on what version of the adblock list it has. Maybe we could include some metadata in the about page so we know exactly what commit the list came from when it was built?

Reply:
@bridiver do we really update this multiple times a day? It's **the list catalog**, not the default list.
https://github.com/brave/adblock-resources/commits/master/

Reply:
> is this really that different than using components in general? 

The components are external things for a browser, the same as extensions or site content.
.pak files are internal things and should be reproducible.

I'm with @bbondy about making an automation if you really need to update this frequently. Chromium also uses this approach.
An example from the upstream: https://skia.googlesource.com/buildbot/+doc/main/autoroll/README.md

Reply:
@atuchin-m the catalog is just the first in the list of things to add from this and yes, the lists do get updated many times a day. In DM we talked about adding an optional gn var to set a specific hash if we wanted an exactly identical build, but in principle this is no different than what happens when the browser downloads the crx

Reply:
@bridiver I believe we need to have reproducible builds by default.

I also don't see any frequent commits in the history that make sense for the client side.

Reply:
why not just stick to the upstream (aka autoroll) approach? we would have a log of changes at least, it's fine to auto commit changes multiple times a day

Reply:
@bridiver
> the catalog is just the first in the list of things to add from this

I believe to add something like the lists is bad idea. It results in excessive memory consumption.
If only you manage to find a way to unload parts of .pak files from a shared memory.

Reply:
I think this discussion is not relevant anymore for this PR because we don't need the entire ad catalog on startup and the parts that we do need very rarely change.

Reply:
@atuchin-m @iefremov are there still concerns with this pointing to master now that it's not part of the Brave binary at all and is just a component that is already different for every install of Brave since it normally gets downloaded on first run?

Reply:
none from me